app/views/posts/show.html.erb

```diff
@@ -1,14 +1,17 @@
-<div class="user_c">
+<div class="user_con">
 <div class="user_data_post"> 
-  <h2>#<%= @post.title %> by <span><%= @post.user.name %></span></h2>
-  <h3 class = "post-comment">Comment <%= @post.comments_counter%>,  <span>Likes <%= @post.likes_counter%></span></h3>
-  <p><%= @post.text %></p>  
+  <h2><%= @post.title %> by <span><%= @post.user.name %></span></h2>
+  <%= form_with url: "/posts/#{@post.id}/likes", scope: :like do |form| %>
+    <%= form.submit "Like ", class: "post-like-btns"%>
+  <% end %>
+  <h3>Comment <%= @post.comments_counter%>,  <span>Likes <%= @post.likes_counter%></span></h3>
```

Comment:
- [ ] Currently, I have added only two comments as you can see below screenshot but it's showing 4. Please fix that.

![Screenshot from 2022-02-25 02-00-01](https://user-images.githubusercontent.com/22434660/155602970-2f10af27-dcee-41c9-9052-4c31c92143ca.png)

Reply:
Thank you for pointing out that error. Fixed this.